src/main.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+from inference import *
+from train import *
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Scene Landmark Detection',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        '--dataset_folder', type=str, required=True,
+        help='Root directory, where all data is stored')
+    parser.add_argument(
+        '--output_folder', type=str, required=True,
+        help='Output folder')
+    parser.add_argument(
+        '--landmark_config', type=str, default='landmarks/landmarks-300',
+        help='Landmark configuration.')
+    parser.add_argument(
```

Comment:
help('File containing information about visibility of landmarks in cameras associated with training set.')